### ISSUE TITLE
gam report domain not work anymore?

### ISSUE THREAD

**Issue by starmood** [jay0lee/GAM#1201]
I used to use 'gam report domain' command to get the license summary information.
But recently I found it does not work anymore, and this command is not listed in document anymore either. 
So is there any other way I can get the overall license information, such as how many license totally I have, and how many license has been used.
I saw there is 'gam print licenses' command, but the output is not like what I needed.

**Comment by jay0lee** [comment]
I suspect you are actually looking for:

```gam info domain```

**Comment by starmood** [comment]
But 'gam info domain' does not give me the license info. 
I am sure 'gam report domain' was working at least 1 week ago.

**Comment by starmood** [comment]
sorry, just found the license info from 'gam info domain'.

So do you know why 'gam report domain' just stopped working recently?

**Comment by taers232c** [comment]
What is your gam version?
--
ross.scroggs@gmail.com

**Comment by jay0lee** [comment]
Sorry, I don't recall ```gam report domain``` ever existing as a command, there is no domain report in the Google APIs. There's also ```gam report customer``` which may be what you were thinking of.

**Comment by taers232c** [comment]
domain and customers are synonyms for customer
```
REPORT_CHOICE_MAP = {
    'access': 'access_transparency',
    'accesstransparency': 'access_transparency',
    'calendars': 'calendar',
    'customers': 'customer',
    'doc': 'drive',
    'docs': 'drive',
    'domain': 'customer',
    'enterprisegroups': 'groups_enterprise',
    'google+': 'gplus',
    'group': 'groups',
    'groupsenterprise': 'groups_enterprise',
    'hangoutsmeet': 'meet',
    'logins': 'login',
    'oauthtoken': 'token',
    'tokens': 'token',
    'usage': 'usage',
    'usageparameters': 'usageparameters',
    'users': 'user',
    'useraccounts': 'user_accounts',
}
```

**Comment by starmood** [comment]
Oh, I just knew they are same command.

And I found I must use latest 5.10 version, and then it returns correct output.
I saw 5.10 fixed some report issues, maybe that's related.

**Comment by starmood** [comment]
Fixed in 5.10
